api_proxy.yaml

```diff
@@ -232,6 +206,32 @@ paths:
           description: Profile cannot be updated.
           schema:
             $ref: "#/definitions/ProblemJson"
+  "/profile/sender-services":
+    x-swagger-router-controller: ServicesController
+    get:
+      operationId: getServicesByRecipient
+      summary: Get Services by recipient
+      description:  |-
+        Returns the service IDs of all the services that have contacted the recipient,
+        identified by the provided fiscal code, at least once.
+      responses:
+        "200":
+          description: Found.
+          schema:
+            $ref: "#/definitions/ServiceList"
+          examples:
+            application/json:
+              page_size: 1
+              items:
+                - service_id: 2b3e728c1a5d1efa035c
```

Comment:
I guess the example here is wrong as it  should contain the service version as well

Reply:
at the moment the example is correct. Will be fixed here: https://www.pivotaltracker.com/story/show/159464415